source/slang/hlsl.meta.slang

```diff
@@ -2484,7 +2484,7 @@ matrix<T, N, M> trunc(matrix<T, N, M> x)
 typedef uint WaveMask;
 
 __target_intrinsic(cuda, "__activemask()")
-WaveMask WaveGetActiveMask() { return 0xffffffff; }
+WaveMask WaveGetConvergedMask() { return 0xffffffff; }
```

Comment:
I'm happy with the rename here. My two questions:

* Do we want to *also* have a `WaveGetActiveMask()` that returns `WaveActiveBallot(true)`? It seems like that would be useful as a parallel accessor, even if it isn't strictly necessary (since HLSL chose to leave out a getter for the active mask, despite it being computable)

* Do we want to make `WaveGetConvergedMask()` return something more "accurate" on targets that don't support masks? I suspect that having it return the same value as `WaveGetActiveMask()` would on targets without more flexible scheduling would be the safest option. In particular it seems like it would be nice to have an invariant that the "converged" mask seen by a thread is always a subset of the "active" mask.